### ISSUE TITLE
Admin panel search doesn't work well

### ISSUE THREAD

**Issue by MrKomish** [ever-co/ever#500]
Currently, field search in tables with pagination seems to the only search in the current results page, while it should search in all the data.

![image](https://user-images.githubusercontent.com/1496352/57387412-f3d41c00-71be-11e9-9f8d-c3535d659bb5.png)

**Comment by evereq** [comment]
@MrKomish yeah, there are lots of issues with `SmartTable` and filtering/sorting, I am not sure it's really working well in all pages, we will need to check and improve. @alish931 do you recall what kind of issues we have? Also maybe you can find a relevant ticket in our JIRA, please?

**Comment by AlishMekliov931** [comment]
Yes, we have a relevant ticket in JIRA: https://everco.atlassian.net/browse/EV-970

**Comment by evereq** [comment]
NOTE: this is a complex issue, we will not work on it for now
